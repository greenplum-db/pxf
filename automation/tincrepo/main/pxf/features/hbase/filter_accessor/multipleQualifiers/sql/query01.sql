-- @description query01 for PXF HBase  - filter accessor - multiple qualifiers

SELECT * from hbase_pxf_with_filter ORDER BY recordkey;