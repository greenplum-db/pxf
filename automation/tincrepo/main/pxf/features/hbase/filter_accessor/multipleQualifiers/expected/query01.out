-- @description query01 for PXF HBase  - filter accessor - multiple qualifiers
SET datestyle TO 'ISO, MDY';
SELECT * from hbase_pxf_with_filter ORDER BY recordkey;
 recordkey |    cf1:q1     |          cf1:q2          | cf1:q3 |     cf1:q4     | cf1:q5 | cf1:q6 | cf1:q7 | cf1:q8 |    cf1:q9     | cf1:q10 |  cf1:q11  |         q12         
-----------+---------------+--------------------------+--------+----------------+--------+--------+--------+--------+---------------+---------+-----------+---------------------
 00000006  | ASCII00000006 | UTF8_計算機用語_00000006 |      7 | lookup00000012 |    6.6 | 6666.6 | &      |      6 | 2160000000006 | t       |   1000000 | 1970-01-01 00:00:36
 00000008  | ASCII00000008 | UTF8_計算機用語_00000008 |      9 | lookup00000016 |    8.8 | 8888.8 | (      |      8 | 5120000000008 | t       | 100000000 | 1970-01-01 00:00:48
(2 rows)

